### ISSUE TITLE
xiaomi.airc.r24r00 gives invalid value `101` for `Filter Life Level`

### ISSUE THREAD

**Issue by Willian-Zhang** [merdok/homebridge-miot#614]
**Describe the bug**
xiaomi.airc.r24r00 gives invalid value when no filter is installed.

**Expected behavior**
disable filter field or shows N/A while its not applicable

**Debug log**
```
[homebridge-miot] This plugin generated a warning from the characteristic 'Filter Life Level': characteristic was supplied illegal value: number 101 exceeded maximum of 100. See https://homebridge.io/w/JtMGR for more info.
```

**Additional context**
``` js
this.addPropertyByString('filter:filter-life-level', '{"siid":21,"piid":1,"type":"urn:miot-spec-v2:property:filter-life-level:0000001E:xiaomi-r24r00:1","description":"Filter Life Level","format":"uint8","access":["read","notify"],"unit":"percentage","valueRange":[0,101,1]}');
    
```
The range is currently autogenerated, more manual override may be required.

**Comment by merdok** [comment]
That is kind of odd as we assume that filter life level is a value between 0 and 100%. What the heck does 101% mean for the filter life level?😅
I guess we would need to manually adjust the auto generated class and set those to 100% max, could you test that out and provide a pull request?

**Comment by Willian-Zhang** [comment]
> "valueRange":[0,100,1]}'

tried this, still generating the warning

**Comment by merdok** [comment]
Then I would need to have a look and handle that in code.

**Comment by merdok** [comment]
Should be fixed with version 1.7.2
